index.html

```diff
@@ -8,7 +8,62 @@
       href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css"
       rel="stylesheet"
     />
-    <title>Document</title>
+    <title>Phispace</title>
   </head>
-  <body></body>
+  <body class="bg-gray-800 text-purple-50">
+    <div class="container flex flex-col items-center w-full mx-auto">
```

Comment:
You could put those classes to the `body` as well, couldn't you?

Reply:
I want to make sure that on fullscreen only a portion of the screen is used. maybe there are more elegant ways?